source/common/access_log/access_log_impl.h

```diff
@@ -19,6 +19,43 @@
 namespace Envoy {
 namespace AccessLog {
 
+/**
+ * Implemention of access logger handles common filter logic.
+ */
+class InstanceImpl : public Instance {
```

Comment:
Let's call this `InstanceBase` to match other places where we provide an abstract base class.

I do really think it belongs in source/extensions/access_loggers/common. I know that's a bit more hassle since it requires a new BUILD file, but it is a common base class for extensions.

Reply:
Went with ImplBase for consistency with the rest of the codebase.